Grbl_Esp32/src/GCode.cpp

```diff
@@ -1473,9 +1473,9 @@ Error gc_execute_line(char* line, uint8_t client) {
             // and absolute and incremental modes.
             pl_data->motion.rapidMotion = 1;  // Set rapid motion flag.
             if (axis_command != AxisCommand::None) {
-                mc_line(gc_block.values.xyz, pl_data);  // kinematics kinematics not used for homing righ now
+                mc_line_kins(gc_block.values.xyz, pl_data, gc_state.position);  // kinematics kinematics not used for homing righ now
```

Comment:
Why does the comment say "kinematics kinematics" ?  Also right is misspelled as righ